linux/system/cpu_frequency.sls

```diff
@@ -0,0 +1,10 @@
+{%- from "linux/map.jinja" import system with context %}
+{%- if system.cpu_frequency is defined %}
+
+ondemand_service_disable:
```

Comment:
I am fine with the use-case, but do it well. If you just need to hack it somehow, use `rc.local`